spec/hadoop_yarn_nodemanager_spec.rb

```diff
@@ -59,8 +59,8 @@
       end.converge(described_recipe)
     end
 
-    it 'ensures /usr/hdp/current/hadoop-yarn-nodemanager/bin/container-executor has proper permissions' do
-      expect(chef_run).to create_file('/usr/hdp/current/hadoop-yarn-nodemanager/bin/container-executor').with(
+    it 'ensures /usr/hdp/2.2.4.2-2/hadoop-yarn/bin/container-executor has proper permissions' do
```

Comment:
It's a constant. The 2.2.4.2 packages will always be 2.2.4.2-2 on disk. This isn't a build revision for the packages, but some internal number out of Hortonworks. If it changes, they're doing something horrible. From what I can tell, any additional version would be **at least** 2.2.4.3-3 on disk.

At any rate, it's a constant. It means that we **must** update the cookbook for every version they release, until they do away with this garbage, if ever. The full path will always point to the correct version, where the current link will always point to the latest version to be installed, even if it's a lower version number.